java/query/src/main/java/sleeper/query/model/output/S3ResultsOutput.java

```diff
@@ -70,25 +77,24 @@ public S3ResultsOutput(InstanceProperties instanceProperties, Schema schema, Map
         String defaultRowGroupSize = instanceProperties.get(DEFAULT_RESULTS_ROW_GROUP_SIZE);
         String defaultPageSize = instanceProperties.get(DEFAULT_RESULTS_PAGE_SIZE);
 
-        this.compressionCodec = config.getOrDefault(COMPRESSION_CODEC, "zstd");
```

Comment:
The compression codec config field is now unused. There's still a static final field for it. Should it be used?

Reply:
Resolved - value should come from the config, and if that's not present then default to the value from the table properties.